contract.md

```diff
@@ -12,29 +14,41 @@ like to work on, as part of your project (for example, other things related to D
 you have come across, or topics from other classes)? Why are you interested in working on
 these things?_
 
+_I want to further work on using extensions because I feel like there were some issues I ran into with using extensions in the last homework assignment, and I feel like extensions have more capabilities than what I used it for. I also want to continuing working on writing an internal langauge because I feel like that applies that most concepts that I've learned from class and homework assignments._
+
 ## Time management plan
 
 _How do you plan to set aside time outside of class, to work on the project? Are there
 intermediate milestones that you can create, to help you make consistent progress?_
 
+_Since the milestones of the project are due every Sunday and the feedback is due every Tuesday, I plan on starting work for the milestones either during or after class on Tuesday, work on it slightly again during or after class on Thursday. Since I have a lot of free time on Fridays, I feel like I'll have time to finish up a lot of work on Fridays. I think I'm mostly going to follow the milestones that are already set for me with the weekly homework assignments._
```

Comment:
I think it's also important to regularly reconsider your milestones to make sure they are feasible while also being challenging as your project progresses and you learn more about the domain.